fennel/connectors/test_connectors.py

```diff
@@ -508,10 +540,38 @@ class UserInfoDataset:
     assert source_request == expected_source_request, error_message(
         source_request, expected_source_request
     )
+
+    sink_request = sync_request.sinks[0]
+    s = {
+        "table": {
+            "kafka_topic": {
+                "db": {
+                    "name": "kafka_src",
+                    "kafka": {
+                        "bootstrap_servers": "localhost:9092",
+                        "security_protocol": "PLAINTEXT",
+                        "sasl_mechanism": "PLAIN",
+                        "sasl_plain_username": "test",
+                        "sasl_plain_password": "test",
+                    },
+                },
+                "topic": "test_topic1",
+                "format": {"json": {}},
+            }
+        },
+        "dataset": "UserInfoDatasetDerived",
+        "dsVersion": 1,
+        "cdc": "Debezium",
+    }
+    expected_sink_request = ParseDict(s, connector_proto.Sink())
+    assert sink_request == expected_sink_request, error_message(
+        sink_request, expected_sink_request
+    )
     sync_request = view._get_sync_request_proto(tier="staging")
-    assert len(sync_request.datasets) == 1
+    assert len(sync_request.datasets) == 2
     assert len(sync_request.sources) == 1
-    assert len(sync_request.extdbs) == 1
+    assert len(sync_request.sinks) == 1
+    assert len(sync_request.extdbs) == 2
```

Comment:
so extdbs here is 2 because we have 1 source (snowflake) for tier = staging and now we have defined a sink (kafka). Can you also ensure that if the same kafka cluster is used in both source and sink, we create a single extdb?